docs/articles/expensify-classic/settings/Set-Notifications.md

```diff
@@ -4,72 +4,66 @@ description: This article is about how to troubleshoot notifications from Expens
-Check to make sure you haven't unintentionally blocked Expensify emails and whitelist [expensify.com](https://community.expensify.com/home/leaving?allowTrusted=1&target=http%3A%2F%2Fexpensify.com%2F), mg.expensify.com, and [amazonSES.com](https://community.expensify.com/home/leaving?allowTrusted=1&target=http%3A%2F%2Famazonses.com%2F) with your email provider.
-
-2. **A "We're having trouble emailing you" banner at the top of your screen**
-Verify that your email address in your account settings is correct and is a real deliverable email address. 
-Re-send Verification Email: Look for an option to re-send a verification email, usually provided when this banner appears.
-
-![ExpensifyHelp_EmailError]({{site.url}}/assets/images/ExpensifyHelp_EmailError.png){:width="100%"}
-
-# Deep Dive 
+Sometimes members may have trouble receiving important email notifications from Expensify, such as Expensify Magic Code emails, account validation emails, secondary login validations, integration emails, or report action notifications (rejections, approvals, etc.). 
+
+# Troubleshooting missing Expensify notifications
+
+## Issue: The email or notification is never received, and no message, banner, or additional context is provided 
+Emails can sometimes be delayed and could take up to 30-60 minutes to arrive in your inbox. If you're expecting a notification that still hasn't arrived after waiting: 
+ - Check your **Email Preferences** on the web via **Settings > Account > Preferences**. In the **Contact Preferences** section, ensure that the relevant boxes are checked for the email type you're missing.
+ - Check your email spam and trash folders, as Expensify messages might end up there inadvertently.
+ - Check to make sure you haven't unintentionally blocked Expensify emails. Allowlist the domains expensify.com, mg.expensify.com, and amazonSES.com with your email provider.
+
+## Issue: A banner that says “We’re having trouble emailing you” shows the top of your screen.
+Confirm the email address on your Expensify account is a deliverable email address, and then click the link in the banner that says "here". If successful, you will see a confirmation that your email was unblocked. 
+
+ ![ExpensifyHelp_EmailError]({{site.url}}/assets/images/ExpensifyHelp_EmailError.png){:width="100%"}
+ 
+ **If unsuccessful, you will see another error:**
+ - If the new error or SMTP message that includes a URL, navigate to that URL for further instructions. 
+ - If the new error or SMTP message that includes "mimecast.com", consult with your company's IT team.
+ - If you see a new error or SMTP message that includes "blacklist", it means your company has configured their email servers to use a third-party email reputation or blacklisting service. Consult with your company's IT team.
```

Comment:
I read this a few times and I think an extra word was included. Let me know if I'm missing something. Also prefer "blocklist" and "allowlist" to "blacklist" and "whitelist. I kept it in the error message because that's the actual message in our code at the moment.

```suggestion
 - If the new error or SMTP message includes a URL, navigate to that URL for further instructions. 
 - If the new error or SMTP message includes "mimecast.com", consult with your company's IT team.
 - If the new error or SMTP message includes "blacklist", it means your company has configured their email servers to use a third-party email reputation or blocklisting service. Consult with your company's IT team.
```